examples/browserify-gulp-example/package.json

```diff
@@ -7,21 +7,25 @@
     "url": "https://github.com/callemall/material-ui.git"
   },
   "scripts": {
-    "start": "gulp"
+    "start": "gulp",
+    "lint": "eslint --ext .js src && echo \"eslint: no lint errors\""
```

Comment:
(PS I realise you grabbed it from the root `package.json`, blame oli, he removed `jsx` :smile: )